features/support/api_helpers.rb

```diff
@@ -597,6 +597,13 @@ def get_supplier_with_reusable_declaration
   JSON.parse(call_api(:get, "/suppliers/#{supplier_framework['supplierId']}").body)['suppliers']
 end
 
+def get_supplier_with_copyable_service(framework)
+  @existing_service = get_a_service("published", framework['family'])
```

Comment:
I saw a bug running this locally where the service found was not present in the list of copyable services in the UI; I think this was because it has `copiedToFollowingFramework: true` (although I can't see the service in the list of draft services for G-Cloud 12 either). We might need some kind of way of filtering out those services.

Reply:
Hmm good shout. I'll take a look

Reply:
Added a commit - it will only look at the first page of 100 published services, but that should (hopefully!) be enough, as this scenario will clean up the copied draft once it's done.